CHANGELOG-5.0.md

```diff
@@ -7,6 +7,8 @@ in 5.0 minor versions.
 To get the diff for a specific change, go to https://github.com/symfony/symfony/commit/XXX where XXX is the change hash
 To get the diff between two versions, go to https://github.com/symfony/symfony/compare/v5.0.0...v5.0.1
 
+* bug #36383 [String] fix multilangue slug replacement with option added 'replace_chars' (lmasforne)
```

Comment:
please revert all changes in this file: it's autogenerated from the title of the PR (please mind the title of the PR :) )

Reply:
Yes i revert this thanks ;)